test_shopcart.py

```diff
@@ -28,6 +28,11 @@ def test_index(self):
         self.assertEqual( resp.status_code, status.HTTP_200_OK )
         self.assertTrue ('Shopcart Demo REST API Service' in resp.data)
 
+    def test_list_shopcarts(self):
+        resp = self.app.get('/shopcarts')
+        self.assertEqual( resp.status_code, status.HTTP_200_OK )
+        self.assertTrue( len(resp.data) > 0 )
```

Comment:
@jiweix Ok so are you suggesting that since we do not have database we can keep this assertion and when we integrate with a database we can change it?